### ISSUE TITLE
mocha api config needs updating

### ISSUE THREAD

**Issue by DrXyzzy** [sagemathinc/cocalc#2415]
Deprecation warning:
```
~/cocalc/src/smc-hub$ node_modules/.bin/mocha --opts test/mocha.opts test/api/basic.coffee

(node:44619) DeprecationWarning: "--compilers" will be removed in a future version of Mocha; see https://git.io/vdcSr for more info
```
As noted in https://git.io/vdcSr in the warning, along with changing the coffeescript option in `mocha.opts`, it's also necessary to add a glob in the `testapi` test directory script in `packages.json`.

**Comment by nemanjacosovic** [comment]
@DrXyzzy Check out the solution > https://github.com/mochajs/mocha/wiki/compilers-deprecation

Everything I needed to do was to switch from --compilers to --require.

**Comment by DrXyzzy** [comment]
fixed in #2416
